_posts/2022-11-25-checking-in-on-dawson.md

```diff
@@ -0,0 +1,57 @@
+
+**Mike Marcotte:** I’m proud of DAWSON’s stability. There have been a few bugs here and there, but compared to the early months after launch, the bugs that we have had to tackle at one time have been few and quickly solved. Deployments are very predictable, and we are marching forward at what seems like a healthy rate. Knock on wood, the application has had only one outage in the last two years due to our service provider’s unexpected outage in one region. DAWSON came out of it more resilient than before. If a similar outage occurred  today, we would stay up and running. We owe much of this to the automated tests that improve with each deployment.
+
+<div class="testimonial-blockquote" markdown=1>
+Every week we think of new opportunities to bring to the system - it’s been great to be able to think outside the box.
+<span>– Stephanie Servoss, Clerk of the Court</span>
+</div>
+
+### _What are some recent challenges or interesting problems that you have been working on?_
+
+**Chris Holly:** Creating the stamp disposition was such an exciting challenge. The Court wanted the digital analogue of those old rubber stamps that you could just “ker-chunk” onto a motion rather than writing an entire order to grant or deny a motion. So we brainstormed with the Court and our industry partner. As we iterated on the design, we shifted from a very context-specific solution to a one-size-fits-all solution that gave the Court enough common options across motions. We realized that we could simplify the mechanism and the change management needed to implement it.  We saw  a willingness among Court stakeholders to collaborate and make process decisions in the interest of delivering the best possible user experience. We ended up with something that covered about 90 percent of the solution, and ultimately has room to grow.
+
+**Mike Marcotte:** Managing a project that relies on open source dependencies has been challenging as some of them stop being maintained. This has forced us to consider alternatives or take on the task of maintaining the dependency ourselves. Specifically, our text editor for drafting documents is no longer supported, and we now are considering using the Court’s Microsoft Azure license to integrate Microsoft Word within the application. Thanks to feature flags, the application can easily be configured to use an alternative solution. Another dependency that is no longer supported is our React state management solution, which is a critical low-level component of the application. We decided to maintain it for now while conducting experiments with other solutions that might better fit the project in the long term.
+
+-----
+
+#### Shout-out and kudos to the many, many team members at the Tax Court, 18F, and industry partners who worked on this project over the years and brought DAWSON to where it is today! :heart:
```

Comment:
style tweak suggestion: let's make this plain paragraph text rather than h4, that will match the style in this sibling blog post: https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.app.cloud.gov/preview/18f/18f.gsa.gov/mgwalker/check-in-with-jerome-lee/2022/11/29/18f-checks-in-with-jerome-lee-and-the-eapd-project/

Reply:
Done! https://github.com/18F/18f.gsa.gov/pull/3629/commits/fdda39735b3708df440ba927cdb0e04d71fc10df